src/tqec/plaquette/rpng_description.py

```diff
@@ -0,0 +1,245 @@
+from tqec.circuit.schedule import ScheduledCircuit
+from tqec.circuit.qubit_map import QubitMap
+from tqec.plaquette.plaquette import Plaquette
+from tqec.plaquette.qubit import PlaquetteQubits
+from tqec.plaquette.qubit import SquarePlaquetteQubits
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+from stim import Circuit as stim_Circuit
+
+
+class BasisEnum(Enum):
+    X = 'x'
+    Y = 'y'
+    Z = 'z'
+
+
+class ExtendedBasisEnum(Enum):
+    X = BasisEnum.X.value
+    Y = BasisEnum.Y.value
+    Z = BasisEnum.Z.value
+    H = 'h'
+
+
+@dataclass
+class RPNG:
```

Comment:
We should group the docstring for attributes of a `dataclass` into the `Attribute` section.